src/routes/boot.ts

```diff
@@ -160,6 +161,34 @@ const setAuthCookie = async (
   return accessToken;
 };
 
+const getAndUpdateLastChangelogRedis = async (
+  con: DataSource,
+): Promise<string> => {
+  let lastChangelogFromRedis = await getRedisObject(REDIS_CHANGELOG_KEY);
+
+  if (!lastChangelogFromRedis) {
+    const post: Pick<Post, 'createdAt'> = await con
+      .getRepository(Post)
+      .findOne({
+        select: ['createdAt'],
+        where: [{ sourceId: 'daily_updates' }],
+        order: {
+          createdAt: {
+            direction: 'DESC',
+          },
+        },
+      });
+
+    if (post) {
+      lastChangelogFromRedis = post.createdAt.toISOString();
+    }
+  }
+
+  await setRedisObject(REDIS_CHANGELOG_KEY, lastChangelogFromRedis);
```

Comment:
This should go inside the `if` so we will not set it constantly

Reply:
Good catch 👌